Source/FieldSolver/SpectralSolver/SpectralKSpace.cpp

```diff
@@ -178,48 +182,65 @@ SpectralKSpace::getModifiedKComponent( const DistributionMapping& dm,
     // Initialize an empty ManagedVector in each box
     KVectorComponent modified_k_comp(spectralspace_ba, dm);
 
-    // Compute real-space stencil coefficients
-    Vector<Real> stencil_coef = getFonbergStencilCoefficients(n_order, nodal);
+    if(n_order == -1){ // Infinite-order case
```

Comment:
Only style suggestions:
```suggestion
    if (n_order == -1) { // Infinite-order case
```